### DISCUSSION TITLE
Abonnement pour smartphone dégooglisé

### DISCUSSION THREAD

**Discussion by MaxCSD** [osmandapp/OsmAnd#20648] [Ideas]
Il y a une possibilité d'avoir un abonnement pour les téléphones sans Google ?

J'ai un Fairephone qui tourne sous /e/ os et je ne peux pas acheter d'application sur le Play Store.

Merci de votre attention !

**Comment by ZeljkoBG** [comment]
Don't you find it strange that everyone writes in English, except you?

**Comment by dmpr0** [comment]
Hi! 
Currently no, all purchases are go thru official stores.
